qa-tests-backend/src/test/groovy/IntegrationsSplunkViolationsTest.groovy

```diff
@@ -134,19 +132,64 @@ class IntegrationsSplunkViolationsTest extends BaseSpecification {
             }
             hasNetworkViolation = results.any { isNetworkViolation(it) }
             hasProcessViolation = results.any { isProcessViolation(it) }
+            // TODO: Remove debug log
+            log.info "Found violations in Splunk: \n${results}"
+            log.info "hasNetworkViolation: ${hasNetworkViolation}\nhasProcessViolation: ${hasProcessViolation}"
+            // TODO: /Remove debug log
             if (hasNetworkViolation && hasProcessViolation) {
                 log.info "Success!"
                 break
             }
         }
 
+        // After we know that violations arrived, manually kick off search that converts them into alerts
+        postToSplunk(port, "/services/saved/searches/" +
+                "Threat%20-%20Create%20Notable%20from%20RHACS%20Alert%20-%20Rule/dispatch", [
```

Comment:
Would that be a good candidate for a const with some description what that URL actually is?